app/controllers/api/internal/v1/api_controller.rb

```diff
@@ -13,7 +13,7 @@ def validate_wca_token
     # Renew our token if it has expired or is close to expiring
     Vault.auth_token.renew_self if vault_token_data[:ttl] < 300
 
-    client = Faraday.new(url: EnvConfig.VAULT_ADDR)
+    client = Faraday.new(url: EnvConfig.VAULT_ADDR, &FaradayConfig)
```

Comment:
If you look into the file we actually parse our own requests here

Reply:
Good catch! Fixed :wink: